### PR TITLE
added serveral endpoints

### PR DESCRIPTION
**Pull request by jasmingacic**
Added routes as requested:

http://147.75.33.177:8080/apis/default/noupstream/definition
http://147.75.33.177:8080/apis/default/noupstream/crd

http://147.75.33.177:8080/staticroutes
http://147.75.33.177:8080/staticroutes/default/todo-frontend/crd
http://147.75.33.177:8080/staticroutes/default/todo-frontend

Fixes https://github.com/kubeshop/kusk-gateway/issues/281 
Fixes https://github.com/kubeshop/kusk-gateway/issues/278

Signed-off-by: Jasmin Gacic <jasmin.gacic@gmail.com>

@topliceanurazvan I got a little lost with all endpoints that need to be added. Can you review and point to which ones are missing?

### CODE REVIEW

**Comment by topliceanurazvan** [comment]
Looks good @jasmingacic , the only endpoint missing is the CRD for the fleets.

**Review by topliceanurazvan — APPROVED** [review]

File: server/go/api_static_routes_service_impl.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	}
+	return Response(http.StatusOK, StaticRouteItem{
+		Name:      staticRoute.Name,
+		Namespace: staticRoute.Namespace,
+	}), nil
+}
+
+// GetStaticRouteCRD - Get static route CRD
+func (s *StaticRoutesApiService) GetStaticRouteCRD(ctx context.Context, namespace string, name string) (ImplResponse, error) {
+	staticRoute, err := s.kuskClient.GetStaticRoute(namespace, name)
```
Comment (kylehodgetts) [inline server/go/api_static_routes_service_impl.go:45]: > staticRoute, err := s.kuskClient.GetStaticRoute(namespace, name)

Should this be called `GetStaticRouteCRD(...)` ?

File: server/go/routers.go [hunk 2]
```diff
@@ -216,4 +216,4 @@ func parseInt32ArrayParameter(param, delim string, required bool) ([]int32, erro
 	}
 
 	return ints, nil
-}
+}
```
Comment (kylehodgetts) [inline server/go/routers.go:219]: newline

**Review by kylehodgetts — CHANGES_REQUESTED** [review]
Some small changes. Could you also `go fmt` all of the code, please? 😁

File: server/README.md [hunk 3]
```diff
@@ -25,20 +25,10 @@ go run main.go
 
 To run the server in a docker container
 ```
-make build
+docker build --network=host -t openapi .
```
[the remark this answers is not in the document]
Comment (jasmingacic) [reply server/README.md:28]: I guess I was not aware of this. change :D 
I will have it reverted

File: server/go/api.go [hunk 4]
```diff
@@ -14,57 +14,81 @@ import (
...
-	GetRawOpenApiSpec(http.ResponseWriter, *http.Request)
 }
-
 // FleetsApiRouter defines the required methods for binding the api requests to a responses for the FleetsApi
 // The FleetsApiRouter implementation should parse necessary information from the http request,
 // pass the data to a FleetsApiServicer to perform the required actions, then write the service results to the http response.
-type FleetsApiRouter interface {
+type FleetsApiRouter interface { 
 	GetEnvoyFleet(http.ResponseWriter, *http.Request)
+	GetEnvoyFleetCRD(http.ResponseWriter, *http.Request)
```
[the remark this answers is not in the document]
Comment (jasmingacic) [reply server/go/api.go:33]: yes, the object as is

File: server/go/api_static_routes_service_impl.go (hunk 1 above)
Comment (jasmingacic) [reply server/go/api_static_routes_service_impl.go:45]: Not really. The only difference is that here we don't do any operations on fleet object we just return it at the end of the method.

Comment (kylehodgetts) [reply server/go/api_static_routes_service_impl.go:45]: Im not sure I understand, the Service method is called `GetStaticRouteCRD` but inside we call the kuskClient `GetStaticRoute`? that's the correct behaviour?

**Review by kylehodgetts — APPROVED** [review]

**Pull request merged**